bait-files/src/main/webapp/submission.js

```diff
@@ -0,0 +1,209 @@
+function loadAmericanAddress() {
+
+    let addressInputDiv = document.getElementById("address-input-div");
+
+    //Address
+    let addressDiv = document.createElement("div");
+    addressDiv.id = "address-div";
+    let addressLabel = document.createElement('label');
+    addressLabel.setAttribute("for", "address-line-input");
+    let addressLabelText = document.createTextNode("Address: ");
+    addressLabel.appendChild(addressLabelText);
+    addressDiv.appendChild(addressLabel);
+    let address = document.createElement("input");
```

Comment:
You need to set the name for each input in order for the name-value pair to be sent with the form. div.setAttribute("name", <name_of_field>).

Reply:
Thank you!  Changed!